vmupdate/agent/source/plugins/manage_rpm_macro.py

```diff
@@ -21,7 +21,7 @@
 
 import os
 
-import pkg_resources
+import packaging
```

Comment:
There is a problem with this. The code in vmupdate/agent directory is copied into VM and run there. This means the dependency on the dom0 package is not enough. And indeed the `python3-packaging` is not installed in a minimal template by default...

Looking for what it's used, maybe it can be dropped and replaced with just `int()` ? or some local helper if any distribution has `X.Y` in the version id field.